.github/workflows/ci.yml

```
@@ -131,7 +133,7 @@ jobs:
     - name: Migrate Database
       run: |
         docker build --target migrate-database -t ilios-migrate-database .
-        docker run --name ilios-migrate-database -e ILIOS_SECRET -e ILIOS_DATABASE_URL -e ILIOS_FILE_SYSTEM_STORAGE_PATH --network=host ilios-migrate-database
+        docker run --name ilios-migrate-database -e ILIOS_SECRET -e ILIOS_DATABASE_URL -e ILIOS_FILE_SYSTEM_STORAGE_PATH -e MESSENGER_TRANSPORT_DSN--network=host ilios-migrate-database
```

```suggestion
        docker run --name ilios-migrate-database -e ILIOS_SECRET -e ILIOS_DATABASE_URL -e ILIOS_FILE_SYSTEM_STORAGE_PATH -e MESSENGER_TRANSPORT_DSN --network=host ilios-migrate-database
```

Looks like maybe a space was missing here and could be the issue.

---

*headdesk. 
thanks @jrjohnson !